DB_Design/Liquibase/liquibase.properties

```diff
@@ -1,6 +1,6 @@
-  changeLogFile: C:\\Users\\Administrator\\LiquibaseMSSQL\\dbchangelog.xml
+  changeLogFile: C:\\Users\\tavar\\Documents\\Workspace\\Github\\Fun_Things\\Fun_Things\\DB_Design\\Liquibase\\dbchangelog.xml
   url: jdbc:sqlserver://localhost:1433;database=shooterdb;
   username: Shooter_app
   password: OMGFun!
   driver: com.microsoft.sqlserver.jdbc.SQLServerDriver
-  classpath: ../../Liquibase_Drivers/mssql-jdbc-7.4.1.jar
\ No newline at end of file
+  classpath: C:\\Users\\tavar\\Documents\\Workspace\\Github\\Fun_Things\\Fun_Things\\DB_Design\\Liquibase\\lib\\Liquibase_Drivers\\mssql-jdbc-7.4.1.jre8.jar
```

Comment:
can we go back to the ../../ format?

Reply:
Will create SOP for local deployment, this will not be needed once in place